public/data/smartCitiesData.json

```diff
@@ -1,7 +1,9 @@
 [
   {
```

Comment:
This is the generated file. It's changed because of changes in `convert.js` and `cities.js`